lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+require 'csv'
+
+class Menu
+  attr_reader :menu_list, :price_list, :dish_list
+
+  def initialize
+    @menu_list = []
+    @price_list = {}
+    @dish_list = []
+    load_menu
+  end
+
+  def print_menu
+    title = "Kiran's cafe menu:"
+    title + @menu_list.to_s
```

Comment:
I would further format this output to include some line breaks and make it read a bit nicer.